requirements.txt

```diff
@@ -7,6 +7,7 @@ dnspython==1.16.0
 graphviz==0.13.2
 humanize==0.5.1
 Jinja2==2.11.3
+markupsafe<2.1 # v2.1 removes code used by jinja2 v2.11.3
```

Comment:
Where's this dependency used?

Reply:
Jinja2 uses markupsafe as part of its library.  Markupsafe 2.1 removes soft_unicode which is used by our version of Jinja2 so the test suite will fail if we don't pin markupsafe to a version that still retains the soft_unicode function.